### DISCUSSION TITLE
prefix and suffix input with inner icon/text

### DISCUSSION THREAD

**Discussion by fira03079508** [filamentphp/filament#3747] [Ideas]
Would be nice that the prefix / prefixIcon / suffix / suffixIcon function would allow you to have a parameter to put the prefix or suffixe inside the input. 

See available options here : https://tailwindui.com/components/application-ui/forms/input-groups

Mostly Input with leading icon and  Input with trailing icon (icon or text)

**Comment by danharrin** [comment]
I would accept a PR to change the existing design

**Reply by fira03079508** [reply]
I don't have much experience with Filament or Tailwind yet. Should I try anyway ?

**Reply by danharrin** [reply]
Yes please, you could try just copying the classes and HTML structure from Tailwind UI
